### PR TITLE
SCP initial

### PR DESCRIPTION
**Pull request by pflynn-virtru**
https://virtru.atlassian.net/browse/PLAT-2297
- Initial commit of Helm charts for SCP

### CODE REVIEW

**Review by c3foto — APPROVED** [review]

File: scp/charts/configuration/templates/redis/persistentvolume.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  capacity:
+    storage: {{ $capacity }}
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: /storage/data{{ . }}
+
+---
+
+{{- end -}}
```
Comment (strantalis) [inline scp/charts/configuration/templates/redis/persistentvolume.yaml:19]: I would leave the storage configuration up to the end user. Creating a persistent volume will require elevated permissions and not everyone will have local storage at `/storage/data`.

If you remove this you won't need the storageclass either.

Comment (pflynn-virtru) [reply scp/charts/configuration/templates/redis/persistentvolume.yaml]: I added `configuration.redis.storage.enabled` so these can be turned off

Comment (strantalis) [reply scp/charts/configuration/templates/redis/persistentvolume.yaml]: I feel like the right approach here is to leave the `PersistentVolume` and `StorageClass` out of this helm chart and leave that up to the cluster administrator to configure the storage options on the cluster. A lot of storage is provisioned dynamically today in kubernetes even on-premise. 

Is there a reason you think these resources would add value to the chart?

Comment (pflynn-virtru) [reply scp/charts/configuration/templates/redis/persistentvolume.yaml]: I'll move the `PersistentVolume` and `StorageClass` Kubernetes YAML to the README

**Pull request merged**